dm/pkg/schema/tracker.go

```diff
@@ -372,6 +372,10 @@ func (tr *Tracker) Reset() error {
 
 // Close close a tracker.
 func (tr *Tracker) Close() error {
+	// prevent SchemaTracker being closed when
```

Comment:
GetTableInfo should use read lock too, to make them mutual exclusive.

Reply:
GetTableInfo has already required RLock currently.